Documentation/ColumnsConfig/CommonProperties/FieldInformation/TcaDescription.rst

```diff
@@ -0,0 +1,105 @@
+..  include:: /Includes.rst.txt
+..  _tca_property_fieldInformation_tcaDescription:
+
+==============
+tcaDescription
+==============
+
+Use this `fieldInformation` to allow adding a more detailed description to the
+form element. `tcaDescription` will read the `description` part of the
+TCA column definition. If that value starts with `LLL:` it will render a
+translated output. Else it will render the text unmodified.
+
+..  include:: /Images/Rst/FieldInformationTcaDescription.rst.txt
+
+..  confval:: tcaDescription
+    :name: fieldInformation-tcaDescription
+    :Path: $GLOBALS['TCA'][$table]['columns'][$field]['config']['fieldInformation']
+    :type: array
+    :Scope: fieldInformation
+
+    ..  note::
+        The HTML tags for this specific area are limited to:
+        `<a>`, `<br>`, `<br/>`, `<div>`, `<em>`, `<i>`, `<p>`, `<strong>`,
+        `<span>`, `<code>`.
+
+`tcaDescription` is activated for most of the TYPO3 form elements by default.
+Currently, following form elements supports `tcaDescription`:
+
+*   BackendLayoutWizardElement.php
```

Comment:
Listing the PHP files which implement the elements in the core is probably not very helpful to developers? We could list all field config names but I would probably ommit it as keeping such a list up to date would be a challenge...

Reply:
I understand the afford to keep such lists up2date. Removed.